source/includes/entityref/_jobshift.md

```diff
@@ -0,0 +1,164 @@
+# JobShift
+
+JobShift is the main entity for shift scheduling and is used for storing start & end times, references to the clientCorporation, job, and other details for the shift (such as location, name, etc).
+
+<table>
+    <colgroup>
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+    </colgroup>
+    <thead>
+        <tr class="header">
+            <th>JobShift fields</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Not null</th>
+            <th>Read-only</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr class="even">
+            <td>id</td>
+            <td>Integer</td>
+            <td>Unique identifier for this entity.</td>
+            <td>X</td>
+            <td>X</td>
+        </tr>
+        <tr class="odd">
+            <td>dateAdded</td>
+            <td>Timestamp</td>
+            <td>Date when this record was created in the Bullhorn system.</td>
+            <td>X</td>
+            <td>X</td>
+        </tr>
+        <tr class="even">
+            <td>dateLastModified</td>
+            <td>Timestamp</td>
+            <td>Date when this record was last modified in the Bullhorn system.</td>
+            <td>X</td>
+            <td>X</td>
+        </tr>
+        <tr class="odd">
+            <td>endTime</td>
+            <td>OffsetDateTime</td>
+            <td>Time the Job Shift Ends.</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr class="even">
+            <td>fillRatio</td>
+            <td>Double</td>
+            <td>Ratio of Assigned Candidates to Openings on the Job Shift.</td>
```

Comment:
Change to:
Ratio of assigned Candidates to Openings on the JobShift.

What is Openings? Is it an actual entity or just a general thing? If general, use lowercase on it.